src/applications/check-in/locales/en/translation.json

```diff
@@ -153,5 +153,17 @@
   "your-date-of-birth-can-not-be-in-the-future": "Your date of birth can not be in the future",
   "please-provide-a-response": "Please provide a response",
   "in-en": "in english",
-  "sorry-we-cant-complete-pre-check-in": "Sorry, we can’t complete pre-check-in"
-}
+  "sorry-we-cant-complete-pre-check-in": "Sorry, we can’t complete pre-check-in",
+  "contact-information": "Contact Information",
+  "a-staff-member-will-help-you-on-the-day-of-your-appointment": "A staff member will help you on the day of your appointment.",
```

Comment:
It looks like this replaces an older key:
'a-staff-member-will-help-you-on-the-day-of-your-appointment-to-update-your-information'

We should probably delete the old key to keep the file a little cleaner

Reply:
Agreed, removed!